diff --git a/CMakeLists.txt b/CMakeLists.txt
index 307af34..c166daa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,7 @@ webos_config_build_doxygen(doc Doxyfile)
 
 # -- no way to disable warn_unused_result right now.
 webos_add_compiler_flags(ALL -g -O3 -Wall -Wno-unused-but-set-variable -Wno-unused-variable -fno-exceptions -fno-rtti -fPIC)
+webos_add_compiler_flags(ALL ${EXTRA_DEFINITIONS})
 webos_add_linker_options(ALL --no-undefined)
 
 # -- add local include paths
diff --git a/Src/Main.cpp b/Src/Main.cpp
index 053c651..d374751 100644
--- a/Src/Main.cpp
+++ b/Src/Main.cpp
@@ -37,8 +37,6 @@
 #include "Settings.h"
 #include "JSONUtils.h"
 
-#include <QApplication>
-
 static void logFilter(const gchar *log_domain, GLogLevelFlags log_level, const gchar *message, gpointer unused_data);
 
 GMainLoop * g_gmainLoop = NULL;
@@ -151,7 +149,6 @@ int main(int argc, char ** argv)
 {
     setenv("QT_PLUGIN_PATH","/usr/plugins",1);
     setenv("QT_QPA_PLATFORM", "minimal",1);
-    QApplication app(argc, argv);
 
 	parseCommandlineOptions(argc, argv);
 	setLoglevel(Settings::settings()->m_logLevel.c_str());
