diff --git a/CMakeLists.txt b/CMakeLists.txt
index 307af34..c166daa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -82,6 +82,7 @@ webos_config_build_doxygen(doc Doxyfile)
 
 # -- no way to disable warn_unused_result right now.
 webos_add_compiler_flags(ALL -g -O3 -Wall -Wno-unused-but-set-variable -Wno-unused-variable -fno-exceptions -fno-rtti -fPIC)
+webos_add_compiler_flags(ALL ${EXTRA_DEFINITIONS})
 webos_add_linker_options(ALL --no-undefined)
 
 # -- add local include paths
