Upstream: submitted

From 4df58b877562659b0853a6e108519b60c5d616c5 Mon Sep 17 00:00:00 2001
From: Nikolay Orlyuk <virkony@gmail.com>
Date: Sat, 16 Feb 2013 16:48:10 +0200
Subject: [PATCH] Correct includes of lunaservice.h

---
 inc/luna/MojLunaDefs.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/inc/luna/MojLunaDefs.h b/inc/luna/MojLunaDefs.h
index a18171e..241a281 100644
--- a/inc/luna/MojLunaDefs.h
+++ b/inc/luna/MojLunaDefs.h
@@ -1,6 +1,6 @@
 /* @@@LICENSE
 *
-*      Copyright (c) 2009-2012 Hewlett-Packard Development Company, L.P.
+*      Copyright (c) 2009-2013 Hewlett-Packard Development Company, L.P.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
@@ -22,7 +22,7 @@
 #define MOJLUNADEFS_H_
 
 #include "core/MojCoreDefs.h"
-#include <lunaservice.h>
+#include <luna-service2/lunaservice.h>
 
 class MojLunaErr;
 class MojLunaMessage;
-- 
1.8.1.2

